src/main/plugin/iso19139.ca.HNAP/layout/layout-custom-fields.xsl

```diff
@@ -293,7 +296,7 @@
     </xsl:call-template>
   </xsl:template>
 
-  <xsl:template mode="mode-iso19139" match="gmd:EX_GeographicBoundingBox" priority="2005">
+  <xsl:template mode="mode-iso19139" match="gmd:EX_GeographicBoundingBox[$schema = 'iso19139.ca.HNAP']" priority="2005">
```

Comment:
This template can be removed, as can rely in the `iso19139` template to handle these elements. Tested working fine.

Reply:
removed